mech_client/cli.py

```diff
@@ -112,6 +118,7 @@ def interact(  # pylint: disable=too-many-arguments
             retries=retries,
             timeout=timeout,
             sleep=sleep,
+            chain_id=chain_id,
```

Comment:
Perhaps this name is confusing. It refers to the ID entry in the mechs.json configuration file (which can be an arbitrary value: polygon, polygon-config-2, etc... )

Reply:
Changed to chain_config